test/e2e/integreatly_test.go

```diff
@@ -460,13 +457,13 @@ func checkOperatorVersions(t *testing.T, f *framework.Framework, namespace strin
 
 	err := f.Client.Get(goctx.TODO(), types.NamespacedName{Name: installationName, Namespace: namespace}, installation)
 	if err != nil {
-		return errors.Wrap(err, "Error getting installation CR from cluster when checking operator versions")
+		return fmt.Errorf("Error getting installation CR from cluster when checking operator versions: %w", err)
 	}
 
 	for product, version := range operatorVersions {
 		clusterVersion := installation.Status.Stages[stage].Products[operator.ProductName(product)].OperatorVersion
 		if clusterVersion != operator.OperatorVersion(version) {
-			return errors.Wrap(err, fmt.Sprintf("Error with version of %s operator deployed on cluster. Expected %s. Got %s", product, version, clusterVersion))
+			return fmt.Errorf("Error with version of %s operator deployed on cluster. Expected %s. Got %s. %w", product, version, clusterVersion, err)
```

Comment:
@grdryn This seems to be the same case as other comments - `err` is irrelevant here.

Reply:
This one "broke" the e2e test -- it was silently succeeding up to now when it should have been erroring. I've added a new commit to fix that, not sure if there will be additional failures after that.